packages/web-components/src/toolbar/toolbar.styles.ts

```diff
@@ -0,0 +1,97 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusMedium,
+  colorNeutralBackground1,
+  colorNeutralForeground1,
+  fontFamilyBase,
+  spacingHorizontalNone,
+  spacingHorizontalS,
+  spacingHorizontalXL,
+  spacingHorizontalXS,
+} from '../theme/design-tokens.js';
+
+/**
+ * Toolbar styles
+ * @public
+ */
+export const styles = css`
+  ${display('inline-flex')}
+
+  :host {
+    --toolbar-item-gap: 4px;
+    align-items: center;
+    background: ${colorNeutralBackground1};
+    border-radius: ${borderRadiusMedium};
+    box-sizing: border-box;
+    color: ${colorNeutralForeground1};
+    font-family: ${fontFamilyBase};
+    height: 40px;
+    padding: ${spacingHorizontalXS} ${spacingHorizontalS};
+    width: 100%;
+  }
+  ::slotted(:not([slot])) {
+    flex: 0 0 auto;
+    flex-direction: row;
+    margin: 0 var(--toolbar-item-gap);
+  }
+
+  .positioning-region {
+    align-items: center;
+    display: inline-flex;
+    flex-flow: row wrap;
+    flex-grow: 1;
+    justify-content: flex-start;
+  }
+  ::slotted([slot='label']) {
+    margin-inline-end: var(--toolbar-item-gap);
+  }
+  ::slotted([slot='start']) {
+    margin: 0 var(--toolbar-item-gap);
+  }
+  ::slotted([slot='end']) {
+    margin-inline-start: auto;
+  }
+  :host([size='small']) {
+    height: 32px;
+    padding: ${spacingHorizontalXS} ${spacingHorizontalNone};
+  }
+  :host([size='large']) {
+    height: 48px;
+    padding: ${spacingHorizontalXS} ${spacingHorizontalXL};
+  }
```

Comment:
The `large` variation should have a bottom border per the Figma spec:
https://www.figma.com/file/QfHyx6ApwFEcw5q4pM9CSZ/Toolbar?type=design&node-id=2367-13918&mode=design&t=yoMeokMOlNQa9cJU-0

<img width="1457" alt="image" src="https://github.com/microsoft/fluentui/assets/5067638/95bef7b9-741b-49b0-9f9d-b88a727c347f">

Reply:
Addressed in new PR